### PR TITLE
Tools

### PR DESCRIPTION
**Pull request by sixtyfourktec**
Some random packages.

### CODE REVIEW

File: recipes/graphics/fonts/dejavu.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+checkoutSCM:
+    scm: url
+    url: ${SOURCEFORGE_MIRROR}/dejavu/${PKG_VERSION}/dejavu-fonts-ttf-${PKG_VERSION}.tar.bz2
+    digestSHA1: 7fa15e7b9676fc3915338c41e76ad454c344fff5
+    stripComponents: 1
+
+buildScript: |
+    FONTS="DejaVuSansMono*.ttf DejaVuSans.ttf DejaVuSans-*.ttf DejaVuSerif.ttf DejaVuSerif-*.ttf DejaVuSansCondensed*.ttf DejaVuSerifCondensed*.ttf"
+    FONTCFG="20-unhint-small-dejavu-sans-mono.conf 57-dejavu-sans-mono.conf 20-unhint-small-dejavu-sans.conf 57-dejavu-sans.conf 20-unhint-small-dejavu-serif.conf 57-dejavu-serif.conf"
```
Comment (jkloetzke) [inline recipes/graphics/fonts/dejavu.yaml:14]: Not enforced yet but could we break such long lines? Also, using arrays (`FONTS=( "DejaVuSansMono*.ttf" ...)`)  would be more bash-like.

Comment (sixtyfourktec) [reply recipes/graphics/fonts/dejavu.yaml]: done

Comment (jkloetzke) [reply recipes/graphics/fonts/dejavu.yaml]: Thanks. But the patterns are now unquoted. Thus, the shell will try apply *pathname expansion*. I guess the patterns won't match but we should play safe here. Also the `${FONTS[@]}` should have quotes: -> `"${FONTS[@]}"` (FONTCFG likewise).

Comment (sixtyfourktec) [reply recipes/graphics/fonts/dejavu.yaml]: done

File: recipes/libs/botan.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+buildToolsWeak: [python3]
+buildVars: [AR, CXX, CXXFLAGS, CPPFLAGS, ARCH]
+buildScript: |
+    mkdir -p build install
+    pushd build
+    $1/configure.py \
+        --cpu=$ARCH \
+        --os=linux \
+        --cxxflags="$CXXFLAGS $CPPFLAGS" \
+        --ldflags="$LDFLAGS" \
```
Comment (sixtyfourktec) [inline recipes/libs/botan.yaml:29]: I checked this and those flags are required so that it finds zlib.

Comment (jkloetzke) [reply recipes/libs/botan.yaml:29]: Ack

**Comment by jkloetzke** [comment]
Thanks

**Pull request merged**